src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IUsingStatement.vb

```diff
@@ -131,32 +156,76 @@ Module Program
         End Sub
     End Class
     Sub Main(args As String())
-        Using c1 As C = New C, c2 As C = New C'BIND:"Using c1 As C = New C, c2 As C = New C"
+        Using c1 As C = New C'BIND:"Using c1 As C = New C"
             Console.WriteLine(c1)
         End Using
     End Sub
 End Module]]>.Value
 
             Dim expectedOperationTree = <![CDATA[
 IUsingStatement (OperationKind.UsingStatement) (Syntax: 'Using c1 As ... End Using')
-  Declaration: 
-    IVariableDeclarationStatement (2 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Using c1 As ... s C = New C')
+  Resources: 
+    IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclarationStatement) (Syntax: 'Using c1 As C = New C')
       IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'c1')
         Variables: Local_1: c1 As Program.C
         Initializer: 
           IObjectCreationExpression (Constructor: Sub Program.C..ctor()) (OperationKind.ObjectCreationExpression, Type: Program.C) (Syntax: 'New C')
             Arguments(0)
             Initializer: 
               null
-      IVariableDeclaration (1 variables) (OperationKind.VariableDeclaration) (Syntax: 'c2')
-        Variables: Local_1: c2 As Program.C
+  Body: 
+    IBlockStatement (1 statements) (OperationKind.BlockStatement) (Syntax: 'Using c1 As ... End Using')
```

Comment:
>IBlockStatement (1 statements) (OperationKind.BlockStatement) (Syntax: 'Using c1 As ... End Using') [](start = 4, length = 99)

It feels like this block should be marked as IsImpicit, consider opening an issue to follow up on that.

Reply:
I don't know as though I agree. Just like the block inside a for loop isn't implicit, I don't know why the block inside a Using statement would be.

---
In reply to: [144369564](https://github.com/dotnet/roslyn/pull/22630#discussion_r144369564) [](ancestors = 144369564)

Reply:
> Just like the block inside a for loop isn't implicit, I don't know why the block inside a Using statement would be.

Doesn't it use the same syntax as its parent?